renderer/components/Syncing/Syncing.js

```diff
@@ -28,13 +29,14 @@ const Syncing = ({
   useEffect(() => {
     setIsWalletOpen(true)
   }, [setIsWalletOpen])
+  const theme = useContext(ThemeContext)
 
   if (isLightningGrpcActive && syncStatus === 'complete') {
     return <Redirect to="/app" />
   }
 
   return (
-    <Panel width={1} {...rest}>
+    <Panel width={1} {...rest} theme={theme}>
```

Comment:
Does Panel really need to be passed `theme`? It seems that it only uses `Box` and `Flex`, both of which should have access to the current theme without needing to be explicitly passed.

Reply:
@mrfelton, same as above, since previously `{...rest}` included the theme, I included it here. I will remove if it's not needed.

Reply:
Done.